### PR TITLE
Added MacOS Support

### PR DESCRIPTION
**Pull request by fabiobaser**
Addded new Binary and MacOS specific Install-Script

### CODE REVIEW

File: n_mac [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+com=""
+for i in `seq $#`
+do
+    com=`echo $com ${!i}`
+done
+$com
+
+osascript -e 'display notification  "Exit code: '"$?"'" with Title "Command \"'"$1"'\" completed"'
```
Comment (madhavanmalolan) [inline n_mac:10]: Can we not put this in the same script with an if-else?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,25 @@
 ![Works on linux](https://img.shields.io/badge/Works%20On-Linux-blue.svg)
+![Works on MacOS](https://img.shields.io/badge/Works%20On-MacOS-blue.svg)
```
Comment (madhavanmalolan) [inline README.md:2]: Thanks for noticing this! :+1:

**Review by madhavanmalolan — CHANGES_REQUESTED** [review]
These changes can be clubbed under an if-else to check platform.

**Comment by madhavanmalolan** [comment]
Closing this instead. Thanks for your contribution :100: 
https://github.com/madhavanmalolan/N/pull/3

File: n_mac (hunk 1 above)
Comment (fabiobaser) [reply n_mac:10]: Sure, i just didn't want to mess with your code at all. :)

File: README.md (hunk 2 above)
Comment (fabiobaser) [reply README.md:2]: Not for that. Easy peasy 👍

**Pull request closed without merging**